### ISSUE TITLE
GODEBUG="sbrk=1" necessary?

### ISSUE THREAD

**Issue by allenluce** [bouk/monkey#3]
I'll sometimes get faults with this kind of backtrace:

```
unexpected fault address 0xc82041c740
fatal error: fault
[signal 0xb code=0x2 addr=0xc82041c740 pc=0xc82041c740]

goroutine 4 [running]:
runtime.throw(0xa1b1c8, 0x5)
        /home/aluce/.gvm/gos/go1.6.2/src/runtime/panic.go:547 +0x90 fp=0xc820037d08 sp=0xc820037cf0
runtime.sigpanic()
        /home/aluce/.gvm/gos/go1.6.2/src/runtime/sigpanic_unix.go:27 +0x2ab fp=0xc820037d58 sp=0xc820037d08
github.com/lyfe-mobile/receiver.StartMovers.func1(0xc8204090e0, 0xc820409140, 0x1312d00, 0xa33b30, 0xa)
        /home/aluce/.gvm/pkgsets/go1.6.2/global/src/github.com/lyfe-mobile/receiver/movers.go:37 +0x531 fp=0xc820037f88 sp=0xc$
20037d58
```

This is at the call site of a patched call. These come intermittently, and seem to do so only after a GC run. I'm not sure what the GC may be doing to mess up the calls, but as a workaround setting `GODEBUG` to `sbrk=1` seems to keep this from happening.

**Comment by allenluce** [comment]
I've opened #4 as one way to avoid having the GC screw stuff up. Works great now!

**Comment by allenluce** [comment]
Fixed with the merge of #4.
